sdk/src/main/java/com/karhoo/sdk/analytics/Payloader.kt

```diff
@@ -62,10 +66,14 @@ class Payloader internal constructor(builder: Builder) {
             return this
         }
 
-        fun addBookingRequest(battery: Float,
-                              connectionType: String,
-                              tripDetails: TripInfo,
-                              outboundTripId: String?): Builder {
+        fun bookingRequested(
+            battery: Float,
+            connectionType: String,
+            tripDetails: TripInfo,
+            outboundTripId: String?,
+            correlationId: String?,
```

Comment:
since this event is fired before we make a call to the backend, there is no correlation id. It will always be passed as null. it should be removed for this reason.